src/components/Chat/UserInput/index.js

```diff
@@ -83,12 +83,9 @@ const UserInput = props => {
         <ButtonContainer className="icon-btn" onClick={() => {}}>
           <div className="send-voice-attachment-button" />
         </ButtonContainer>
-        <input
-          type="submit"
-          value="➤"
-          className="send-message-button"
-          tabIndex="0"
-        />
+        <button type="submit" className="send-message-button" tabIndex="0">
```

Comment:
ButtonContainer automatically sets type as button and not submit, which I need here. I guess that could be changed so that type is passed as props but at least I couldn't do it quickly so that I wouldn't need to set type for all buttons using ButtonContainer..

Reply:
Well that makes sense! :) Let's leave it this way so there is no need to change all the other buttons now.